lib/osx.js

```diff
@@ -50,9 +50,10 @@ var OSXBuilder = {
     options.log( 'Kicking off ´appdmg´' );
 
     var appdmg = require( 'appdmg' );
+    var fileName = options.config.osx.fileName ? options.config.osx.fileName : options.config.osx.title;
```

Comment:
Can we add a line break after `?` and `:` here.
I'm a fan of the 80 characters per line if possible. :blush: